src/lib/rnp2.c

```diff
@@ -97,6 +98,34 @@ struct rnp_output_st {
     bool                 keep;
 };
 
+struct rnp_op_sign_st {
+    rnp_ffi_t    ffi;
+    rnp_input_t  input;
+    rnp_output_t output;
+    rnp_ctx_t    rnpctx;
+};
+
+struct rnp_op_verify_signature_st {
+    uint8_t      keyid[PGP_KEY_ID_SIZE];
+    char         hash_fn[16];
```

Comment:
Can you not use a `pgp_hash_alg_t` here? (mostly don't like seeing the arbitrary size)

Reply:
Sure, done.